helm/chart/maesh/charts/metrics/values.yaml

```diff
@@ -4,6 +4,10 @@ image:
   configmapReload: jimmidyson/configmap-reload:v0.2.2
 
 grafana:
+  service:
+    type: ClusterIP
+    port: 3000
+  # nodePort: null
```

Comment:
Please document the `nodePort` field, like other commented fields are documented.

Reply:
Thanks for the review, updated to include comments for nodePort